compiler/src/iree/compiler/Reducer/Framework/WorkItem.h

```diff
@@ -0,0 +1,41 @@
+// Copyright 2023 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef IREE_COMPILER_REDUCER_WORK_ITEM_H
+#define IREE_COMPILER_REDUCER_WORK_ITEM_H
+
+#include "mlir/IR/Builders.h"
+#include "mlir/IR/BuiltinOps.h"
+
+namespace mlir {
+namespace iree_compiler {
```

Comment:
Note here and elsewhere: better to have a dedicated namespace (and can use compact form): `namespace mlir::iree_compiler::Reducer {`